.travis.yml

```diff
@@ -34,4 +31,5 @@ script:
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_snmp_information
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_users
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_traceroute
+- nosetests -v TestNXOSDriver:TestNXOSDriver_bgp_neighbors
```

Comment:
Why are you using setUp vs setUpClass? Just trying to understand what you are doing to see if that's something we need in the new framework.